### ISSUE TITLE
Comments support

### ISSUE THREAD

**Issue by olayway** [flowershow/flowershow#44]
Would be nice not to render user comments.

**Flowershow:**
![image](https://user-images.githubusercontent.com/52197250/179369099-feb15584-cf63-4f02-ab3b-f3dda252fc53.png)

**Obsidian preview:**
![image](https://user-images.githubusercontent.com/52197250/179369113-ba661696-72c1-4421-acae-b2db2b28e13b.png)

**Comment by rufuspollock** [comment]
Given we have MDX you could just use html/jsx commenting which mdx supports i believe see e.g. https://github.com/mdx-js/mdx/issues/184

My suggestion here is we just add a page to document this e.g. `docs/comments`

**Comment by sahilm-10** [comment]
@rufuspollock @olayway Can you guide me ? I want to contribute. I am new to Opensource

**Comment by rufuspollock** [comment]
@sahilm-10 what do you understand of the issue for yourself?

**Comment by rufuspollock** [comment]
@sahilm-10 yes i think the start is to research docs support in mdx and then add a page docs/comments to flowershow.app exclaiming how to add comments to pages ...
